packages/odyssey-react/src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  display: none;
+
+  &.isOpen {
+    display: block;
+  }
+}
+
+.overlay {
+  display: flex;
+  position: fixed;
+  z-index: 10;
+  inset-block-start: 0;
+  inset-inline-end: 0;
+  inset-block-end: 0;
+  inset-inline-start: 0;
+  align-items: center;
+  justify-content: center;
+  background: rgba(cv('gray', '900'), 0.75);
+
+  .root[aria-hidden='false'] & {
```

Comment:
Good cleanup from the previous implementation

Reply:
We can do even better now that we have a component and JS! These types of nested and referential selectors with higher specificity can be flattened out and simplified by applying a single class to the `dialog` and `overlay` elements that trigger these animations.

Reply:
Will get to this as part of OKTA-419315!